modin/logging/config.py

```diff
@@ -83,14 +86,17 @@ def bytes_int_to_str(num_bytes, suffix="B"):
         Human-readable string format.
     """
     factor = 1000
+    n_bytes: float = num_bytes
     for unit in ["", "K", "M", "G", "T", "P"]:
-        if num_bytes < factor:
-            return f"{num_bytes:.2f}{unit}{suffix}"
-        num_bytes /= factor
-    return f"{num_bytes * 1000:.2f}P{suffix}"
+        if n_bytes < factor:
+            return f"{n_bytes:.2f}{unit}{suffix}"
+        n_bytes /= factor
+    return f"{n_bytes * 1000:.2f}P{suffix}"
```

Comment:
Why is this change necessary? Can we instead make `num_bytes` a float at the function signature and change the docstring? I'm just not a fan of `n_bytes: float = num_bytes`.

Reply:
If I were to leave it as is, I'd get an error like this:
```
modin/logging/config.py:93:9: error: Incompatible types in assignment (expression has type "float", variable has type "int")  [assignment]
```

One alternative solution might be to do something like: 
```python
n_bytes = num_bytes # type: float
```
This is a little janky workaround and seems like a semantic difference. Maybe if we wanted to be more explicit we could do something like:
```python
n_bytes = float(num_bytes)
```

Does anyone else have any thoughts about this? @modin-project/modin-contributors

Reply:
Agreed with @mvashishtha.